.packit.yaml

```diff
@@ -101,6 +101,25 @@ jobs:
           # builds older than the final release
           - bash -c "hatch version | sed -E 's/\\.[0-9]+\\.dev/.dev/'"
 
+  # Release to copr
+  - job: copr_build
+    trigger: release
+    targets:
+      - fedora-all
+      - epel-9
+    enable_net: False
+    list_on_homepage: True
+    preserve_project: True
+    owner: psss
```

Comment:
could we have this under `teemtee` group rather, it seems a leftover after moving to `teemtee` org

Reply:
Yeap, makes sense. The new `teemtee` group requested: https://pagure.io/fedora-infrastructure/issue/11547

Reply:
Will be covered, together with docs in https://github.com/teemtee/tmt/pull/2358.